cb_maintenance/views/maintenance_request.xml

```diff
@@ -90,6 +132,38 @@
                 <attribute name="create">false</attribute>
                 <attribute name="delete">false</attribute>
             </kanban>
+            <field name="name" position="after">
+                <field name="maintenance_type"/>
+            </field>
+            <xpath expr="//div[hasclass('o_dropdown_kanban')]/.." position="attributes">
+                <attribute name="t-attf-style">background-color:#{record.maintenance_type.raw_value == 'corrective'? '#ffe0d4': '#daffd4'}</attribute>
+            </xpath>
+            <xpath expr="//div[hasclass('o_dropdown_kanban')]" position="attributes">
+                <attribute name="t-if">1==0</attribute>
+            </xpath>
+        </field>
+    </record>
+
+    <record model="ir.ui.view" id="maintenance_request_manager_kanban_view">
+        <field name="name">maintenance.request.kanban (in cb_maintenance for manager)</field>
+        <field name="model">maintenance.request</field>
+        <field name="inherit_id" ref="maintenance_request_kanban_view"/>
+        <field name="groups_id" eval="[(4, ref('maintenance.group_equipment_manager'))]"/>
+        <field name="arch" type="xml">
+            <kanban position="attributes">
+                <attribute name="create">true</attribute>
+            </kanban>
+        </field>
+    </record>
+
+    <record model="ir.ui.view" id="hr_equipment_request_view_search">
+        <field name="name">maintenance.request.kanban (in cb_maintenance for manager)</field>
+        <field name="model">maintenance.request</field>
+        <field name="inherit_id" ref="maintenance.hr_equipment_request_view_search"/>
+        <field name="arch" type="xml">
+            <filter name="my_requests" position="attributes">
+                <attribute name="domain">['|', '|', ('technician_user_id', '=', uid), ('manager_id', '=', uid), ('owner_user_id', '=', uid)]</attribute>
+            </filter>
```

Comment:
Create two more
 - "On my hands": [('follower_id', '=', uid)]
 - "Managed by me": [('manager_id', '=', uid)]
also, this fields should be searchable